src/components/LaunchList/query.ts

```diff
@@ -1,7 +1,7 @@
 import gql from 'graphql-tag';
 
 export const QUERY_LAUNCH_LIST = gql`
-  query LaunchList {
+  query LaunchListQuery {
     launches {
       flight_number
```

Comment:
Run `yarn start`  then try to remove this line with `flight_number` for instance and TypeScript starts complaining

```sh
Failed to compile.

/Users/alex/dev/graphql-react-typescript-spacex/src/components/LaunchList/LaunchList.tsx
TypeScript error: Property 'flight_number' does not exist on type 'LaunchListQuery_launches'.  TS2339

    24 |                 key={i}
    25 |                 className={`${className}__item`}
  > 26 |                 onClick={() => handleIdChange(launch.flight_number!)}
       |                                                      ^
    27 |               >
    28 |                 {launch.mission_name} ({launch.launch_year})
    29 |               </li>
```